apps/raw_data_check

```diff
@@ -91,14 +91,12 @@ def run(args):
         sys.exit('ERROR: Invalid data path %s' % data_path)
 
     for f in files:
-        raw_data = load_data(f)
+        raw_data = load_neuron(f)
```

Comment:
Are ```load_data``` and ```load_neuron``` the same through the morphio api (i.e. loading of the original file with warts and all)?